src/views/account/Homeworks/Homeworks.tsx

```diff
@@ -88,17 +88,26 @@ const WeekView: Screen<"Homeworks"> = ({ route, navigation }) => {
   const firstDateEpoch = dateToEpochWeekNumber(firstDate);
 
   // Function to get the current week number since epoch
-  const getCurrentWeekNumber = () => {
+  const getWeekNumber = () => {
     const now = new Date();
     now.setHours(0, 0, 0, 0);
     const start = new Date(1970, 0, 0);
     start.setHours(0, 0, 0, 0);
     const diff = now.getTime() - start.getTime();
     const oneWeek = 1000 * 60 * 60 * 24 * 7;
-    return Math.floor(diff / oneWeek);
+    let weekNumber = Math.floor(diff / oneWeek);
+
+    // Check if it's Saturday (6) or Sunday (0)
+    // If so, return the following week
+    const dayOfWeek = now.getDay();
+    if (dayOfWeek === 6 || dayOfWeek === 0) {
```

Comment:
```suggestion
    if (dayOfWeek >= 5 || dayOfWeek === 0) {
```
J'pense que tu peux le faire dès le vendredi, on suppose que l'élève ne fait pas les devoirs le jour même ;)

Reply:
Ah ben laisse comme c'était
Mais jspas, faut voir ce que fait Pronote

Reply:
Ok non en fait, après réflexion, c'est mieux à partir du samedi

Reply:
Tu peux mettre en resolved!

Reply:
y a des gens qui ont cours le samedi !